examples/error_handling.py

```diff
@@ -0,0 +1,80 @@
+import discord
+from discord.ext import commands
+import traceback
+
+description = '''
+A basic bot featuring error handling for command errors
+'''
+
+# Constructing the bot with a prefix and the description
+bot = commands.Bot(command_prefix='!', description=description)
+
+# Example command to showcase a potential error raised.
+# The important ones to see are `MissingRequiredArgument` and `BadArgument`.
+# `MissingRequiredArgument` will be raised when a required argument is not passed.
+# Bad Argument will be raised in this case when the arg cannot be converted to an `int`.
+@bot.command()
+async def add(ctx, arg1: int, arg2: int):
+    added = arg1 + arg2
+    await ctx.send('{} + {} = {}'.format(arg1, arg2, added))
+
+# This command showcases the `is_owner` check, which will raise `commands.NotOwner` if the user doesn't own the bot.
+@commands.is_owner()
+@bot.command()
+async def owner_check(ctx):
+    await ctx.send('{}, you do own this bot!'.format(ctx.author))
```

Comment:
```suggestion
    await ctx.send('{}, you own this bot!'.format(ctx.author))
```
Weird additional word?